conf/dataset.jsonld

```diff
@@ -59,33 +58,20 @@
          "hbz"
       ]
    },
-   "isBasedOn": {
-     "type": "Dataset",
-     "url": "https://d-nb.info/gnd/7749153-1",
-     "name":{
-        "de":"Gemeinsame Normdatei (GND)",
-        "en":"Integrated Authority File (GND)"
-     },
-     "description":{
-        "de":"Genutzt werden die von der DNB bereitgestellten RDF-Datenabzüge der GND, siehe <a href=\"https://www.dnb.de/DE/Service/DigitaleDienste/LinkedData/linkeddata_node.html\">https://www.dnb.de/DE/Service/DigitaleDienste/LinkedData/linkeddata_node.html</a>.",
-        "en":"lobid-gnd uses the RDF dumps of the GND as provided by DNB, see <a href=\"https://www.dnb.de/EN/Service/DigitaleDienste/LinkedData/linkeddata_node.html\"></a>."
-     },
-     "license": "https://creativecommons.org/publicdomain/zero/1.0/"
-   },
    "distribution":[
       {
-         "id":"https://lobid.org/gnd/search",
+         "id":"https://rpb.lbz-rlp.de/search",
          "type":"WebAPI",
          "name": {
-           "de": "lobid-gnd-API",
-           "en": "lobid-gnd API"
+           "de": "RPPD-API",
+           "en": "RPPD API"
          },
          "description": {
          "en": "<p>The API gives access to machine-readable data (JSON-LD via HTTP).</p>",
          "de": "<p>Die API bietet Zugriff auf strukturierte Daten (JSON-LD via HTTP).</p>"
          },
-         "url":"https://lobid.org/gnd/search",
-         "documentation": "https://lobid.org/gnd/api",
+         "url":"https://rpb.lbz-rlp.de/search",
+         "documentation": "https://rpb.lbz-rlp.de/api",
```

Comment:
These URLs fo not work for me as well. Strangely, it shows me lobid-gnd URLs in the UI (alongisde other GND references, partly already mentioned above) at https://rppd.lobid.org/dataset#! :

![image](https://github.com/user-attachments/assets/422547e4-f14d-4925-b536-bb221aea8b83)

Reply:
Forget it, I did look at production not test, where everything is fine: https://rppd-test.lobid.org/dataset#!